src/main/protobuf/tensorflow/core/framework/config.proto

```diff
@@ -1,132 +0,0 @@
-syntax = "proto3";
```

Comment:
This was from tensorflow.core.protobuf, which is not used currently. I will add it back with device placement support.